tests/NevarTest.php

```diff
@@ -40,7 +41,6 @@ public function provideVariables()
             [new \stdClass(), 'object of class stdClass'],
             [new FakeClass(), 'object of class Thunder\Nevar\Tests\Fake\FakeClass'],
             [unserialize('O:1:"A":0:{}'), 'object of class __PHP_Incomplete_Class'],
-            [function () {}, 'object of class Closure'],
```

Comment:
@peter279k why is this test case removed?

Reply:
Hi @thunderer, thank you for your reply. I think it's my fault to remove this test case.

I will add this test case back.

Reply:
No problem, thanks for contributing! I'm happy to merge it now. :)